python/vyos/utils/list.py

```diff
@@ -18,3 +18,26 @@ def is_list_equal(first: list, second: list) -> bool:
     if len(first) != len(second) or len(first) == 0:
         return False
     return sorted(first) == sorted(second)
+
+
+def list_strip(l: list, k: list, right: bool = False) -> list:  # noqa: E741
+    """Remove list k from beginning (right=False), resp., end of list l"""
```

Comment:
```suggestion
def list_strip(lst: list, sub: list, right: bool = False) -> list:
    """Remove list `sub` from beginning (`right=False`), resp., end of list `lst`"""
```

In my opinion `l` and `k` are not descriptive name of parameters